src/pages/using-spark/components/button.mdx

```diff
@@ -198,28 +187,80 @@ Use when data is being saved or submitted. The spinning indicator will replace t
   titleAttr="Spinning Tertiary Button Example"
 />
 
+<SprkDivider element="span" additionalClasses="sprk-u-Measure"></SprkDivider>
 
-### Full Width at Smaller Viewports
+## Button Groups
 
-At small viewport sizes, a Button may be used at full width.
+Buttons can be grouped with other buttons to establish a relationship between them.
 
-- Stacked Buttons should have the primary action above secondary actions.
-- Use full width Buttons in containers that take up 1/3 of the grid or smaller.
+- Group buttons based on their usage and importance.
+- A max of two buttons should be in a group.
```

Comment:
"Use no more than two buttons in a group."
or
"Group no more than two buttons together."

Reply:
![image](https://user-images.githubusercontent.com/4342363/114437995-35348980-9b95-11eb-8678-b32508e479d3.png)

Because our gatsby is weird when we use indented lists, it adds space after the last item. I would change the bullet point with sub-items to be the last in this list, or do it like we do in Color Usage table of contents.